docs/dg/dev/upgrade-and-migrate/upgrade-to-php-83.md

```diff
@@ -0,0 +1,94 @@
+---
+title: 'Upgrade to PHP 8.3'
+description: End of October 2024, Spryker will release a new version of its Demo Shops supporting PHP 8.3.
+template: concept-topic-template
+last_updated: Nov 1, 2024
+---
+
+This guide provides step-by-step instructions to upgrade your project to use PHP 8.3 in both your Docker environment and `composer.json`. 
+By setting the minimum PHP version requirement to 8.3, you ensure that your project fully utilizes the latest features and improvements of PHP 8.3. 
+
+## 1. Check project code for PHP compatibility using PHPCompatibility
+
+To ensure your project is compatible with PHP 8.3, perform the following steps:
+
+1. Require the `php-compatibility` package:
+
+```bash
+composer require --dev phpcompatibility/php-compatibility --ignore-platform-reqs
+```
+
+2. Configure `phpcs` to use `php-compatibility` rules:
+```bash
+vendor/bin/phpcs --config-set installed_paths vendor/phpcompatibility/php-compatibility
+```
+
+3. Execute the `php-compatibility` sniffer for PHP 8.3:
+
+```bash
+vendor/bin/phpcs -p src/ --standard=PHPCompatibility  --runtime-set testVersion 8.3
+```
+
+This returns the code that's not compatible with PHP 8.3.
+
+4. Fix all discovered incompatibilities.
+
+## 2. Check for Incompatible Dependencies
+
+Identify any dependencies that are not compatible with PHP 8.3:
+
+```bash
+composer why-not php 8.3
+```
+
+## 3. Update composer.json
+
+Modify your `composer.json` file to reflect the new PHP version requirements.
+
+1. Set the minimum required PHP version to 8.3.
+
+```bash
+    "require": {
+      "php": ">=8.3",
+    }
+``` 
+
+2. Ensure Composer resolves dependencies compatible with PHP 8.3:
+
+```bash
+  "config": {
+    "preferred-install": "dist",
+    "platform": {
+      "php": "8.3.12"
+    },
+```
+
+3. Run the following command to update the `composer.lock` file:
+
+```bash
+  composer update --lock
```

Comment:
None of those steps actually update the dependencies.
Please at least mention that the identified dependencies should be updated.